gulp-assets/dash-core-components/R/dccChecklist.R

```diff
@@ -0,0 +1,18 @@
+# AUTO GENERATED FILE - DO NOT EDIT
+
+dccChecklist <- function(id=NULL, options=NULL, value=NULL, className=NULL, style=NULL, inputStyle=NULL, inputClassName=NULL, labelStyle=NULL, labelClassName=NULL, loading_state=NULL, persistence=NULL, persisted_props=NULL, persistence_type=NULL) {
+    
+    props <- list(id=id, options=options, value=value, className=className, style=style, inputStyle=inputStyle, inputClassName=inputClassName, labelStyle=labelStyle, labelClassName=labelClassName, loading_state=loading_state, persistence=persistence, persisted_props=persisted_props, persistence_type=persistence_type)
+    if (length(props) > 0) {
+        props <- props[!vapply(props, is.null, logical(1))]
+    }
+    component <- list(
+        props = props,
+        type = 'Checklist',
+        namespace = 'dash_core_components',
+        propNames = c('id', 'options', 'value', 'className', 'style', 'inputStyle', 'inputClassName', 'labelStyle', 'labelClassName', 'loading_state', 'persistence', 'persisted_props', 'persistence_type'),
+        package = 'dashCoreComponents'
```

Comment:
@HammadTheOne Given the reasons outlined in https://github.com/plotly/dashR/pull/243#discussion_r633575595, I believe we'll need to retain the `package` key and update it to reflect the migration of the underlying JS into the `dash`package.

So the JSON returned here should look like this, since the namespace will remain the same, but `package` will change (so R can identify the proper system path when Dash serves up the asset to the browser):

```
{
  "props": {},
  "type": "Div",
  "namespace": "dash_html_components",
  "propNames": ["children", "id", "n_clicks", "n_clicks_timestamp", "key", "role", "accessKey", "className", "contentEditable", "contextMenu", "dir", "draggable", "hidden", "lang", "spellCheck", "style", "tabIndex", "title", "loading_state"],
  "package": "dash"
} 
```